version.props

```diff
@@ -1,65 +0,0 @@
-﻿<Project>
-  <PropertyGroup>
-    <MajorVersion>3</MajorVersion>
-    <MinorVersion>0</MinorVersion>
-    <PatchVersion>0</PatchVersion>
-    <PreReleaseLabel>preview</PreReleaseLabel>
-    <OfficialBuildId Condition="'$(OfficialBuildId)' == ''">$(BUILD_BUILDNUMBER)</OfficialBuildId>
-  </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(OfficialBuildId)' != '' ">
-    <!-- This implements core versioning. Spec: https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Versioning.md -->
-    <_BuildNumber>$(OfficialBuildId)</_BuildNumber>
-
-    <!-- _BuildNumber from CI is assumed to have format "yyyyMMdd.r". -->
-    <_BuildNumberYY>$(_BuildNumber.Substring(2, 2))</_BuildNumberYY>
-    <_BuildNumberMM>$(_BuildNumber.Substring(4, 2))</_BuildNumberMM>
-    <_BuildNumberDD>$(_BuildNumber.Substring(6, 2))</_BuildNumberDD>
-    <_BuildNumberR>$(_BuildNumber.Substring(9))</_BuildNumberR>
-
-    <!-- yy * 1000 + mm * 50 + dd -->
-    <_BuildNumberShortDate>$([MSBuild]::Add($([MSBuild]::Add($([MSBuild]::Multiply($(_BuildNumberYY), 1000)), $([MSBuild]::Multiply($(_BuildNumberMM), 50)))), $(_BuildNumberDD)))</_BuildNumberShortDate>
-
-    <VersionSuffixBuildOfTheDay>$([System.Convert]::ToInt32($(_BuildNumberR)))</VersionSuffixBuildOfTheDay>
-
-    <_BuildNumberSuffix>$(_BuildNumberShortDate).$(VersionSuffixBuildOfTheDay)</_BuildNumberSuffix>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <_BuildNumberSuffix Condition=" '$(_BuildNumberSuffix)' == '' ">0</_BuildNumberSuffix>
-    <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
-    <VersionSuffix>$(PreReleaseLabel).$(_BuildNumberSuffix)</VersionSuffix>
-
-    <ExperimentalVersionPrefix>0.1.$(PatchVersion)</ExperimentalVersionPrefix>
-
-    <!-- Servicing builds have different characteristics for the way dependencies, baselines, and versions are handled. -->
-    <IsServicingBuild Condition=" '$(PreReleaseLabel)' == 'servicing' ">true</IsServicingBuild>
```

Comment:
We need this property still. It's used by reference resolution in servicing builds. Can you keep this code? It can live with the other version variables.

Reply:
It's now in Versions.props